src/01-general/user-invites.apib

```diff
@@ -32,3 +32,19 @@ Cancel a user invite.
         + id: `e88131bf-50c4-49d3-8ae3-47e5d9626bf6` (string, required)
 
 + Response 204
+
+### userInvites.resend [GET /userInvites.resend]
+
+Resend a user invite by canceling the previous invite and creating a new one.
```

Comment:
I thought we used US English but looks like we should use British English. So "cancelling" was correct!

```suggestion
Resend a user invite by cancelling the previous invite and creating a new one.
```